qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -489,6 +489,8 @@ def backends(
         instance: Optional[str] = None,
         dynamic_circuits: Optional[bool] = None,
         filters: Optional[Callable[[List["ibm_backend.IBMBackend"]], bool]] = None,
+        *,
+        use_fractional_gates: Optional[bool] = True,
```

Comment:
Can you add this parameter to the `backend()` method as well? I think many people use that instead of `backends()` when they know the name of the backend.